rmf_robot_sim_common/src/slotcar_common.cpp

```diff
@@ -276,6 +279,8 @@ void SlotcarCommon::handle_ackermann_path_request(
   AckermannTrajectory traj(
     Eigen::Vector2d(locations[0].x, locations[0].y),
     Eigen::Vector2d(locations[1].x, locations[1].y));
+  if (locations[1].obey_approach_speed_limit)
+    traj.approach_speed_limit = locations[1].approach_speed_limit;
```

Comment:
`approach_speed_limit` should be check that it is larger than 0, as it looks like it is not checked further down the code path, https://github.com/open-rmf/rmf_simulation/blob/feature/speed_limits/rmf_robot_sim_common/src/utils.cpp#L32